source/openapi-admin-v3.yaml

```diff
@@ -7109,6 +7109,19 @@ components:
                     can process events in parallel. If `false`, event
                     ordering is enabled and the Trigger executes events
                     serially.
+            error_handler:
+              type: object
+              description: An object that defines custom error handling for an AWS EventBridge trigger. Valid only for `AWS_EVENTBRIDGE` event processors. 
```

Comment:
Is it worth adding a note (or adjusting the current note) that this is only valid for Database triggers as well?

Currently we support AWS EventBridge processors on all of our trigger types (Database, scheduled, auth), but that isn't out long term plan -- we plan to only support eventbridge for Database triggers eventually. So with that future in mind, we only implemented support for Custom Error Handlers with Database triggers.

Reply:
This object is only defined in the database type schema, but it wouldn't hurt to reiterate it here, as well. I'll update

Also, I meant to ask: is this programmatically enforced as invalid for the other trigger types?

Reply:
Yes, it is enforced by the api